awx/main/utils/handlers.py

```diff
@@ -21,27 +22,31 @@ def _connect_unixsocket(self, address):
         super(RSysLogHandler, self)._connect_unixsocket(address)
         self.socket.setblocking(False)
 
+    def handleError(self, record):
+        # for any number of reasons, rsyslogd has gone to lunch;
+        # this usually means that it's just been restarted (due to
+        # a configuration change) unfortunately, we can't log that
+        # because...rsyslogd is down (and would just put us back down this
+        # code path)
+        # as a fallback, it makes the most sense to just write the
+        # messages to sys.stderr (which will end up in supervisord logs,
+        # and in containerized installs, cascaded down to pod logs)
+        # because the alternative is blocking the
+        # socket.send() in the Python process, which we definitely don't
+        # want to do)
+        msg = f'{record.asctime} ERROR rsyslogd was unresponsive: '
```

Comment:
By default, this function in cpython `logging.Handler.handleError()` prints a full traceback and is **extremely** noisy.  This version cuts the failure reporting down to 1-2 lines per emit.

Reply:
https://github.com/python/cpython/blob/v3.8.5/Lib/logging/__init__.py#L987-L1032